pkg/cmd/cli/cmd/delete.go

```diff
@@ -0,0 +1,103 @@
+package cmd
```

Comment:
Why are you reimplementing delete?  This should not be in our codebase, but fixes made upstream and then we can patch them in our code.